docs/logs/pipelines.md

```diff
@@ -0,0 +1,82 @@
+
+By default, we will load only 10 documents but you can also change it to 50 or 100. Note that larger number of documents may slow preview a bit down since documents are sent for processing on each pipeline configuration change.
+
+#### Processors
+Processors are units of processing in pipelines. They can change, drop or even produce additional events. Their are chained to form a pipeline. The output of one processor is the input for the next processor.
+Configuration section will be empty in most cases when you initially open pipeline builder.
+
+![Empty Pipeline](../../images/logs/pipelines/empty-pipeline.png)
+
+In such cases, instead of selected processor configuration, there will be list of available processors. Click on any processor card will add processor with default values. Processors can also be added by clicking on "Add Processor" button.
+
+![Add Processor](../../images/logs/pipelines/add-processor.png)
+
+When clicked on the button, a modal with a searchable list of processors will appear to select processor to add.
+You can also add processor by duplicating some of existing processors.
+
+![Processor Actions](../../images/logs/pipelines/processor-actions.png)
+
+Duplicate action is under our standard action button in the list of processors. Note that order of processors matter since the output of one processor is the input for the next processor. You can reorder processor using drag handle left from the processor's name.
+
+Selected processor configuration will be displayed on the right side. Every processor has at least name and description fields, but all processors will also have additional, processor specific configurations.
+
+##### Errors
+![Processor Errors](../../images/logs/pipelines/processor-errors.png)
+
+In case processor is not properly configured, nuber of errors will be displayed next to processor name and `Save Changes` button will be disabled.
+
+##### Filters
+Almost all processors have optional filters. With filters, you can select only a subset of events that the processor should be applied to. Events that do not match filters will be ignored.
+
+![Processor Filters](../../images/logs/pipelines/processor-filters.png)
+
+It uses our standard filter component with difference that you can also enter field that is not a keyword or even not yet defined. Another important difference is that you can use wildcards for values. It supports standard wildcards `*`, matches any number of characters, and `?`, matches a single character.
+This way you can e.g. drop events that have message field `*password*`.
+
+##### Sampling
+Events sampling is possible using Pipeline. It can be applied on all events or filters can be used to sample only some subset of events, e.g. only events from some service. Sample rate is expressed as percentage of events that should be stored. By default it is 100%, meaning no events will be dropped. Setting sampling to 10% means that only 1 in 10 events will be stored and 9 will be dropped.
```

Comment:
```suggestion
You can do events sampling with Pipeline as well. It can be applied on all events, or by using filters you can sample only a subset of events. One example might be to filter by events from one specific service. The sample rate is expressed as a percentage of events that should be stored. By default, it is 100%, meaning no events will be dropped. Setting sampling to 10% means that only 1 in 10 events will be stored and 9 will be dropped.
```